cmd/copyEnumeratorHelper.go

```diff
@@ -13,10 +13,13 @@ import (
 var EnumerationParallelism = 1
 var EnumerationParallelStatFiles = false
 
+// TODO: potentially remove addTransfer and replace this with scheduleCopyTransfer
 // addTransfer accepts a new transfer, if the threshold is reached, dispatch a job part order.
 func addTransfer(e *common.CopyJobPartOrderRequest, transfer common.CopyTransfer, cca *CookedCopyCmdArgs) error {
-	// Remove the source and destination roots from the path to save space in the plan files
-	transfer.Source = strings.TrimPrefix(transfer.Source, e.SourceRoot.Value)
+	// Remove the destination roots from the path to save space in the plan files
+
+	// TODO: Remove this code because transfer.Source will already have relative path
+	// transfer.Source = strings.TrimPrefix(transfer.Source, e.SourceRoot.Value)
```

Comment:
is transfer.Destination not a similar case? Does this bug end up existing on the destination?

Reply:
Good call out! I removed the transfer.Destination (in the new draft PR) as well because transfer.Destination will always have relative path.